src/analyze/analyze.c

```diff
@@ -82,6 +86,9 @@ static const char *arg_root = NULL;
 STATIC_DESTRUCTOR_REGISTER(arg_dot_from_patterns, strv_freep);
 STATIC_DESTRUCTOR_REGISTER(arg_dot_to_patterns, strv_freep);
 
+static char *dependency_color[_UNIT_DEPENDENCY_MAX] = { NULL };
+static char *dependency_style[_UNIT_DEPENDENCY_MAX] = { NULL };
```

Comment:
please drop the NULL thing. Yes, C99 requires to have at least one item betwwen the `{` and the the `}`, but it doesn't make much semantical sense, and gcc/llvm all don't actually require it, and that's generally what we hack against

Reply:
dropped